README.md

```diff
@@ -79,38 +349,51 @@ The same model can be trained via command line:
 DeezyMatch -i ./inputs/input_dfm.yaml -d dataset/dataset-string-similarity_test.txt -m test001
 ```
 
-A new model directory called `test001` will be created in `models` directory (as specified in the `models_dir` in the input file).
+A new model directory called `test001` will be created in `models` directory (as specified in the input file, see `models_dir` in `./inputs/input_dfm.yaml`).
 
 :warning: Dataset (e.g., `dataset/dataset-string-similarity_test.txt` in the above command)
 * Currently, the third column (label column) should be one of (case-insensitive): ["true", "false", "1", "0"]
 * Delimiter is fixed to `\t` for now.
 
 DeezyMatch keeps some information about the metrics (e.g., loss/accuracy/precision/recall/F1) for each epoch. It is possible to plot the log-file by:
 
+```python
+from DeezyMatch import plot_log
+
+# plot log file
+plot_log(path2log="./models/test001/log.txt", 
+         dataset="t001")
+```
+
+or:
+
 ```bash
-DeezyMatch -lp ./models/test001/log.txt -ld test001
+DeezyMatch -lp ./models/test001/log.txt -ld t001
 ```
 
 In this command, 
-* `-lp`: runs the log plotter
+* `-lp`: runs the log plotter on `./models/test001/log.txt` file. 
 * `-ld` is a name assigned to the log which will be used in the figure. 
```

Comment:
We should make clearer that `-lp` is the same as `path2log`, and `-ld` is the same as `dataset` (but we should probably change the name of this argument)

Reply:
I agree. `-ld` is not the best name. I didn't change the `-ld` flag yet, but changed the function argument.

Reply:
OK, changed it to `"-lo", "--log_output_name"`, I am testing now but will push the changes in few mins.